x-pack/test_serverless/api_integration/test_suites/common/elasticsearch_api/home.ts

```diff
@@ -6,14 +6,28 @@
  */
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
-
+import { RoleCredentials } from '../../../../shared/services';
 export default function ({ getService }: FtrProviderContext) {
-  const esSupertest = getService('esSupertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc: RoleCredentials;
 
   describe('Home', function () {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     it('can request /', async () => {
-      const { body, status } = await esSupertest.get('/');
+      const { body, status } = await supertestWithoutAuth
+        .get('/')
+        .set(svlCommonApi.getCommonRequestHeader())
+        .set(roleAuthc.apiKeyHeader)
+        .redirects(2);
```

Comment:
Question: do we need `redirects` after we switch to API key?

Reply:
In this case yes.  FYI, I'm merging these pr's like you suggested mate.